SPECS/systemd/systemd.spec

```diff
@@ -1,35 +1,14 @@
-Summary:        Systemd-250
+Summary:        Systemd
 Name:           systemd
-Version:        250.3
-Release:        19%{?dist}
+Version:        254.5
+Release:        1%{?dist}
 License:        LGPLv2+ AND GPLv2+ AND MIT
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          System Environment/Security
 URL:            https://www.freedesktop.org/wiki/Software/systemd/
-Source0:        https://github.com/%{name}/%{name}-stable/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
-Source1:        50-security-hardening.conf
-Source2:        systemd.cfg
-Source3:        99-dhcp-en.network
-Source4:        99-mariner.preset
-Patch0:         fix-journald-audit-logging.patch
-# Patch1 can be removed once we update systemd to a version containing the following commit:
-# https://github.com/systemd/systemd/commit/19193b489841a7bcccda7122ac0849cf6efe59fd
-Patch1:         add-fsync-sysusers-passwd.patch
-# Patch2 can be removed once we update systemd to a version containing the following commit:
-# https://github.com/systemd/systemd/commit/d5cb053cd93d516f516e0b748271b55f9dfb3a29
-Patch2:         gpt-auto-devno-not-determined.patch
-# Patch3 can be removed once we update to major version 251 or higher:
-Patch3:         CVE-2022-3821.patch
-# Patch4 can be removed once we update to version 252
-Patch4:         CVE-2022-45873.patch
-Patch5:         backport-helper-util-macros.patch
-Patch6:         CVE-2022-4415.patch
-Patch7:         serve-stale-0001-resolved-added-serve-stale-feature-implementation-of.patch
-Patch8:         serve-stale-0002-resolved-Initialize-until_valid-while-storing-negati.patch
-# Patch9 should be dropped for mariner 3
-Patch9:         mariner-2-do-not-default-zstd-journal-files-for-backwards-compatibility.patch
-Patch10:        update-cifs-for-kernel-headers-6.1.patch
+Source0:        https://github.com/%{name}/%{name}-stable/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+Source1:        95-disable-systemd-oomd.preset
```

Comment:
Does this need to be in `systemd.signatures.json` too?

Reply:
I am not actually clear on why the signatures json is needed at all, but it seems like the toolkit only needs the source tarball included in it.

Reply:
All source files need to be in the signatures.json file. When the toolkit attempts to pack the SRPM files it verifies all of the sha256 hashes of each source. some notes here https://github.com/microsoft/CBL-Mariner/blob/42bc30b1ad4745468911e6749b460947c5e3e9bc/toolkit/docs/building/building.md?plain=1#L295

Reply:
What I'm not really clear on is what benefit the signature file actually brings...it doesn't add any security to anything, it just seems to add extra work? For example, a malicious actor who is able to change one of the files included in the srpm build can just as easily update the hash in the signatures file...there is no added security benefit.